pupil_src/shared_modules/marker_tracker_3d/localization.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+
+from marker_tracker_3d import utils
+
+
+class Localization:
```

Comment:
Merge with CameraLocalizer and give camera_model as direct argument instead of storage.